apps/designer/app/canvas/shared/use-shortcuts.ts

```diff
@@ -29,7 +29,7 @@ type HandlerEvent = {
 };
 
 const togglePreviewMode = () => {
-  publish({ type: "togglePreviewMode" });
+  isPreviewModeStore.set(!isPreviewModeStore.get());
```

Comment:
=== false

Reply:
For me in `if` it makes sense, but in inverting value my mind is not working.
like `setOpen(open === false)` look really hard to follow